src/newsletter-editor/testing/index.js

```diff
@@ -26,48 +26,48 @@ export default compose( [
 			savePost,
 		};
 	} ),
-] )( ( { apiFetchWithErrorHandling, inFlight, postId, savePost, setInFlightForAsync } ) => {
-	const [ testEmail, setTestEmail ] = useState(
-		window?.newspack_newsletters_data?.user_test_emails?.join( ',' ) || ''
-	);
+] )(
+	( { apiFetchWithErrorHandling, inFlight, postId, savePost, setInFlightForAsync, ...props } ) => {
+		const { testEmail, onChangeEmail } = props;
```

Comment:
Non-blocking nitpick: why not destructure these variables on the previous line along with the other props?

Reply:
You're right! I wanted to have them together for legibility but doesn't look like we have a problem with having them together with the other component props.

Reply:
Got it, looks good now!